products/compute/api.yaml

```diff
@@ -7914,7 +7914,127 @@ objects:
             description: |
                 All instances with these tags will be mirrored.
             item_type: Api::Type::String
- 
+
+  - !ruby/object:Api::Resource
+    name: 'PerInstanceConfig'
+    base_url:  'projects/{{project}}/zones/{{zone}}/instanceGroupManagers/{{instance_group_manager}}'
+    min_version: beta
+    description: |
+      A config defined for a single managed instance that belongs to an instance group manager. It preserves the instance name
+      across instance group manager operations and can define stateful disks or metadata that are unique to the instance.
+    create_verb: :POST
+    create_url: projects/{{project}}/zones/{{zone}}/instanceGroupManagers/{{instance_group_manager}}/createInstances
+    update_verb: :POST
+    update_url: projects/{{project}}/zones/{{zone}}/instanceGroupManagers/{{instance_group_manager}}/updatePerInstanceConfigs
+    delete_verb: :POST
+    delete_url: projects/{{project}}/zones/{{zone}}/instanceGroupManagers/{{instance_group_manager}}/deletePerInstanceConfigs
+    read_verb: :POST
+    self_link: projects/{{project}}/zones/{{zone}}/instanceGroupManagers/{{instance_group_manager}}/listPerInstanceConfigs
+    identity:
+      - name
+    nested_query: !ruby/object:Api::Resource::NestedQuery
+      keys:
+        - items
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation': 'https://cloud.google.com/compute/docs/instance-groups/stateful-migs#per-instance_configs'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/beta/instanceGroupManagers'
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/zones/{{zone}}/operations/{{op_id}}'
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
+          insert_minutes: 6
```

Comment:
createInstances i assume will need a little more time here? or on deletion

Reply:
I haven't seen it take very long, but happy to bump these up